dev/Dockerfile

```diff
@@ -57,9 +62,7 @@ RUN dnf -y install yum-utils epel-release.noarch && \
 
 RUN curl https://rclone.org/install.sh | bash
 
-ARG GIT_CLONE_ARGS="--depth 1 https://github.com/rucio/rucio.git"
```

Comment:
Had a look and we're not assigning this ARG in any workflow, so I think this change of removing it is fine (I don't see use cases where we might want to overwrite this)